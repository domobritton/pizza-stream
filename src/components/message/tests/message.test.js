import React from 'react';
import renderer from 'react-test-renderer';
import Message from '../';

describe('Message', () => {
  test('snapshot', () => {
    expect(
      renderer
        .create(<Message />)
    ).toMatchSnapshot();
  });
});
