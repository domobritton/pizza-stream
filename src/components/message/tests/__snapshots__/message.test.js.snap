// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message snapshot 1`] = `
<section
  className="css-1pz3047"
>
  <h2
    className="css-1r6zqto"
  >
    Thanks for your order!
  </h2>
  <div
    className="css-tbnkrp"
  >
    See below for the status of your pizza.
  </div>
</section>
`;
